R/variable.R

```diff
@@ -205,6 +205,17 @@ unbind <- function (x) {
     invisible(out)
 }
 
+#' @rdname webApp
+#' @export
+#' @importFrom utils browseURL
+setMethod("webApp", "CrunchVariable", function (x) browseURL(APIToWebURL(x)))
+
+setMethod("APIToWebURL", "CrunchVariable", function (x) {
+    url <- gsub("(app.crunch.io\\/api\\/datasets)", "app.crunch.io/dataset", self(x))
```

Comment:
1) I'd just match on `/api/datasets/` instead of including `app.crunch.io` in it since the domain won't always be `app` (e.g. profiles, but think also alpha, test01, etc.).
2) Seems like this part should be the same as what the `CrunchDataset` method is. I mean, logically it must be, given the frontend URL scheme. So it would make sense if it used the same code.